chain/app/db_app.go

```diff
@@ -241,7 +242,32 @@ func (app *dbBandApp) EndBlock(req abci.RequestEndBlock) (res abci.ResponseEndBl
 	if err != nil {
 		panic(err)
 	}
-	// Do other logic
+
+	events := res.GetEvents()
+	for _, event := range events {
+		if event.Type == zoracle.EventTypeRequestExecute {
+			var requestID int64
+			var resolveStatus string
+			var result []byte
+			for _, kv := range event.Attributes {
+				if string(kv.Key) == zoracle.AttributeKeyRequestID {
+					requestID, err = strconv.ParseInt(string(kv.Value), 10, 64)
+					if err != nil {
+						break
```

Comment:
If you serious about parse requestID should use `panic`, but if you want to ignore this request it should be `continue`

In this case, my opinion is using `panic`